app/assets/stylesheets/style.css

```diff
@@ -59,4 +59,8 @@ footer.page-footer {
 .social-media-logo {
   height:50px;
   width:50px;
+}
+
+.padding-top {
+  padding-top: 15px;
 }
```

Comment:
@husseinagrama  FYI...because the way the Rails assets pipeline is configured...this change might not take effect in production until we run `rake assets:precompile`.